services/payments/tests/unit/conftest.py

```diff
@@ -24,15 +26,72 @@
     PaymentInitiated,
 )
 
+#
+# rabbit-MQ
+#
+
 
 @pytest.fixture
 def disable_rabbitmq_and_rpc_setup(mocker: MockerFixture) -> Callable:
-    def _doit():
-        # The following moduls are affected if rabbitmq is not in place
+    def _do():
```

Comment:
callable could also work, might sound a bit more close to the language of python